mmdet/models/utils/norm.py

```diff
@@ -1,17 +1,25 @@
 import torch.nn as nn
 
-norm_cfg = {'BN': nn.BatchNorm2d, 'SyncBN': None, 'GN': None}
+norm_cfg = {'BN': nn.BatchNorm2d, 'SyncBN': None, 'GN': nn.GroupNorm}
 
 
 def build_norm_layer(cfg, num_features):
     assert isinstance(cfg, dict) and 'type' in cfg
     cfg_ = cfg.copy()
-    cfg_.setdefault('eps', 1e-5)
     layer_type = cfg_.pop('type')
 
+    # args name matching
+    if layer_type == 'GN':
+        assert 'num_groups' in cfg
+        cfg_.setdefault('num_channels', num_features)
+    elif layer_type == 'BN':
+        cfg_ = dict()  # rewrite neccessary info for BN from here
```

Comment:
This may lead to potential bugs, since `cfg` may contain keys like `momentum`, `affine`, etc.

```python
def build_norm_layer(cfg, num_features):
    assert isinstance(cfg, dict) and 'type' in cfg
    cfg_ = cfg.copy()
    cfg_.setdefault('eps', 1e-5)
    layer_type = cfg_.pop('type')

    if layer_type not in norm_cfg:
        raise KeyError('Unrecognized norm type {}'.format(layer_type))
    elif norm_cfg[layer_type] is None:
        raise NotImplementedError
    else:
        norm_layer = norm_cfg[layer_type]

    if layer_type != 'GN':
        return norm_layer(num_features, **cfg_)
    else:
        assert 'num_groups' in cfg_
        return norm_layer(num_channels=num_features, **cfg_)
```